config/configgrpc/configgrpc.go

```diff
@@ -62,6 +57,26 @@ type GRPCSettings struct {
 	WaitForReady bool `mapstructure:"wait_for_ready"`
 }
 
+// TLSConfig exposes client TLS configuration.
+type TLSConfig struct {
+	// Root CA certificate file for TLS credentials of gRPC client. Should
+	// only be used if `secure` is set to true.
+	CaCert string `mapstructure:"cert_pem_file"`
```

Comment:
The config property name hasn't been changed.

Can we make a breaking change and change it to include `ca` in name? e.g `ca_cert_pem_file`? Or just `ca_cert`?

Reply:
We can probably do that in a separate PR.

This is a breaking change when it comes to the code (not configuration) so you will need to followup with a PR in the collector-contrib where you update otel version (using `make`, see the rule there) and update all the code. Also need to make a quick announcement on the gitter, so if there are others who depend on this they know about this change.

Reply:
@dmitryax updated OTEL collector in the contrib in PR https://github.com/open-telemetry/opentelemetry-collector-contrib/pull/220/files#diff-993db3ecf423549b9fcb5b7e4d195efeR10. 

That corresponds to https://github.com/open-telemetry/opentelemetry-collector/commit/80ee56387e3473bf0cc70a749441948055cf3682 that landed to master after this PR.

Reply:
@bogdandrutu ping me if there is anything I can do more. The refactoring/unifying of the TLS spec will be done in #933